crates/bevy_pbr/src/light/spot_light.rs

```diff
@@ -7,23 +7,78 @@ use super::*;
 #[derive(Component, Debug, Clone, Copy, Reflect)]
 #[reflect(Component, Default)]
 pub struct SpotLight {
+    /// The color of the light.
+    ///
+    /// By default, this is white.
     pub color: Color,
+
     /// Luminous power in lumens, representing the amount of light emitted by this source in all directions.
     pub intensity: f32,
+
+    /// Range in meters that this light illuminates.
+    ///
+    /// Note that this value affects resolution the shadow maps; generally, the
```

Comment:
```suggestion
    /// Note that this value affects resolution of the shadow maps; generally, the
```

I think the intention was?